security/entry_point.rst

```diff
@@ -33,23 +33,25 @@ You can configure this using the ``entry_point`` setting:
     .. code-block:: php
 
         // config/packages/security.php
-        use App\Security\SocialConnectAuthenticator;
-        use Symfony\Config\SecurityConfig;
-
-        return static function (SecurityConfig $security): void {
-            $security->enableAuthenticatorManager(true);
-            // ....
+        namespace Symfony\Component\DependencyInjection\Loader\Configurator;
 
-            // allow authentication using a form or HTTP basic
-            $mainFirewall = $security->firewall('main');
-            $mainFirewall
-                ->formLogin()
-                ->customAuthenticators([SocialConnectAuthenticator::class])
+        use App\Security\SocialConnectAuthenticator;
 
-                // configure the form authentication as the entry point for unauthenticated users
-                ->entryPoint('form_login');
-            ;
-        };
+        return App::config([
+            'security' => [
+                'firewalls' => [
+                    'main' => [
+                        // allow authentication using a form or a custom authenticator
+                        'form_login' => null,
+                        'custom_authenticators' => [
+                            SocialConnectAuthenticator::class,
+                        ],
+                        // configure the form authentication as the entry point for unauthenticated users
+                        'entry_point' => 'form_login',
+                    ],
+                ],
+            ],
+        ]);
```

Comment:
The keys `form_login` and `custom_authenticators` does not exist in the configuration. The issue is already in the Yaml.

Reply:
can you open an issue about this?